docs/examples/perf_analyzer/README.md

```diff
@@ -0,0 +1,46 @@
+# perf_analyzer example with DALI image decoder
+
+
+## Run the example
+
+### tritonserver
+
+To run the tritonserver, first you need to serialize DALI pipeline and put it inside model repo.
+
+`setup_perf_analyzer_example.sh` is a convenience script, that automatizes setting up.
+Provided you have DALI wheel installed in your system, you can just call `sh ./setup_perf_analyzer_example.sh`
```

Comment:
I retained the `sh setup_perf_analyzer_example.sh`. I don't want to set `+x` mod for the `setup...sh` file. So that's the way you should actually run it.